src/sentry/shared_integrations/response/xml.py

```diff
@@ -0,0 +1,9 @@
+from bs4 import BeautifulSoup
+
+from sentry.shared_integrations.response.base import BaseApiResponse
+
+
+class XmlApiResponse(BaseApiResponse):
```

Comment:
Is the logic behind these refactors that each class should have its own file? Even small ones like this? Not disagreeing at all, just wanting to understand the thought process.

Reply:
The reason for this PR is so that you can be more granular in `mypy.ini` and other tools that only take files as parameters. The only time I'd be in favor of combing class definitions into a single file would be if each class is trivially simple and require a lot of imports that aren't worth duplicating.